### ISSUE TITLE
[training issue] Train a model from scratch instead of using pre-trained model

### ISSUE THREAD

**Issue by Rahn80643** [eric612/MobileNet-YOLO#235]
Hi, 
Currently I'm trying to follow the setting of expansion rate which is mentioned in MobileNetv2 paper, so I've trained a model from scratch with my dataset. The mAP of the model is 0.803, but it cannot detect objects well compared to the model trained with default settings and a pretrained model.

Also, I've tried to train a MobileNetv2-YOLO model from scratch, and its mAP is 0.79. Compare the model trained from scratch and a previous model trained with pretrained model, some objects still cannot be detected by using the model trained from scratch.

If I want to train a model from scratch, is there other hyperparameters apart from training batchsize, max iteration can be modified, and is there other techniques to improve the accuracy ?
